apps/mobile/src/libs/components/tag/tag.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+import {
+	LinearGradient,
+	Planet,
+	Text,
+	View,
+} from "~/libs/components/components";
+import { colorToGradientColors } from "~/libs/components/planet/libs/maps/maps";
```

Comment:
Since this map is used not only in Planet component, move it to ~/libs/maps/maps